cnames_active.js

```diff
@@ -1486,6 +1486,7 @@ var cnames_active = {
   "keyframe-refiner": "keyframe-refiner.github.io/keyframe-refiner",
   "keypress": "rumkin.github.io/keypress.js.org",
   "keystone": "keystone-ssg.netlify.app",
+  "keyvhq": "microlinkhq.github.io/keyhq",
   "keyv": "microlinkhq.github.io/keyv",
```

Comment:
maybe it should be pointed to `microlinkhq.github.io/keyhq` too?

```suggestion
  "keyv": "microlinkhq.github.io/hq",
```

Reply:
You can't have two subdomains pointing to a single repo because GitHub only supports a single CNAME per repo.